EarthBnB/pom.xml

```diff
@@ -15,17 +16,31 @@
 	<description>Demo project for Spring Boot</description>
 	<properties>
 		<java.version>11</java.version>
+		<flyway.user>root</flyway.user>
+		<flyway.password>root</flyway.password>
+		<flyway.url>jdbc:mysql://localhost:3306/earthdb</flyway.url>
```

Comment:
add this because for some reason flyway clean would not work until I added them (even if I used the command line args for the same info).